### DISCUSSION TITLE
Opacity bug

### DISCUSSION THREAD

**Discussion by charlesroddie** [wieslawsoltes/Svg.Skia#183] [Q&A]
Svg content:
```
<svg viewBox="30 0 870 1050">
<path fill="#ffffffff"
    d="M702 960c-54.2 52.6-114 44.4-171 19.6-60.6-25.3-116-26.9-180 0-79.7 34.4-122 24.4-170-19.6-271-279-231-704 77-720 74.7 4 127 41.3 171 44.4 65.4-13.3 128-51.4 198-46.4 84.1 6.8 147 40 189 99.7-173 104-132 332 26.9 396-31.8 83.5-72.6 166-141 227z
    M423 237C414.9 113 515.4 11 631 1c15.9 143-130 250-208 236z"/>
</svg>
```
Display on a black background:
![image](https://github.com/wieslawsoltes/Svg.Skia/assets/19760720/8ab83e34-b4de-4f93-96d8-b2199431a94e)

With "#ffffff", it correctly displays as:
![image](https://github.com/wieslawsoltes/Svg.Skia/assets/19760720/dbc7c8c8-12bd-40eb-a279-3a48e8b576a2)

**Comment by wieslawsoltes** [comment]
Default fill is black

**Reply by charlesroddie** [reply]
The bug has `fill="#ffffffff"` so why is the default relevant?

**Reply by dbriard** [reply]
Got the same issue today, when the fill color has transparency (alpha component), it is rendered as black, even if FF is used.

**Reply by wieslawsoltes** [reply]
The alpha value is not supported in hex color string

**Reply by dbriard** [reply]
Hum too bad, a limitation of SkiaSharp? Thx for your quick reply!

**Reply by dbriard** [reply]
I think I found a workaround here: https://shopify.github.io/react-native-skia/docs/images-svg/

![image](https://github.com/wieslawsoltes/Svg.Skia/assets/3595443/1e4b2d61-6cc2-41a9-9e11-6d5f61aef07f)

I'll try to search and replace the svg file at loading time.

**Reply by dbriard** [reply]
With fill-opacity, it render correctly, but unfortunatly it do not work for hit-test.

It is for use in a design project, do you think transparency supports and hit-testing could be something that you can fix when you have free time?

![image](https://github.com/wieslawsoltes/Svg.Skia/assets/3595443/f5d09f22-995d-4283-983c-47238fa87933)

**Reply by wieslawsoltes** [reply]
You can contact me and get support contract if needed, but next year I have fully booked.

**Reply by dbriard** [reply]
Thank you for helping, I found a workaround of workaround... when I add fill-opacity: 0.002, hit test works, and the surface seems to look fully transparent.
